app/controllers/checkouts_controller.rb

```diff
@@ -181,5 +179,11 @@ def checkin_bak
       end
     end
   end
+
+  # def reply
```

Comment:
An alternative is to move the commented out code to a new feature branch.  That way we don't have commented out code in master but the WIP code is still around for work next year.